drivers/lora/sx12xx_common.c

```diff
@@ -359,6 +385,7 @@ int sx12xx_init(const struct device *dev)
 	dev_data.events.RxDone = sx12xx_ev_rx_done;
 	/* TX timeout event raises at the end of the test CW transmission */
 	dev_data.events.TxTimeout = sx12xx_ev_tx_timed_out;
+	dev_data.events.RxError = sx12xx_ev_rx_error;
```

Comment:
Can this be moved under the assignment to `RxDone`, just to keep the RX callbacks together.

Reply:
Done